### DISCUSSION TITLE
Features outstanding and new ideas

### DISCUSSION THREAD

**Discussion by Electrotechnique** [ElectroTechnique/TSynth-Teensy4.1#81] [T4.1 code development]
- Oscilloscope needs improvement
- Keytracking never seemed correctly implemented
- USB client or host port mass storage device, to save and install patches onto pc/USB key
- Will the volume knob adjust anything other than headphone volume?
- Improve SD card response - this could be from just moving to TD1.54, which has improved SD card library
- Better filter - some ladder filter models are available, but the Type control needs to do something

**Comment by winder** [comment]
The volume mixer doesn't seem to do anything, but removing it introduced some strange crackling sounds.
I have my eye on it to allow for multi-timbral volume overrides.

**Reply by Electrotechnique** [reply]
The volume knob? It only affects headphone volume. There was a debate with myself and others as to what it should be doing. The USB digital audio and line out shouldn't (in my opinion) change volume, so the knob only adjusts headphone volume. It could be changed to adjust the VolumeMixer object, which was the intention originally.
